src/findcdn/findcdn.py

```diff
@@ -167,7 +167,9 @@ def interactive() -> int:
                 None,
                 And(
                     str,
-                    lambda filename: os.path.isfile(filename),
+                    lambda filename: os.path.isfile(  # pylint: disable=unnecessary-lambda
```

Comment:
Why do you need a lambda here?  Why can't you just use `os.path.isfile(filename)` directly?

Reply:
It was more of a "I am going to try to keep this here since it was working before". Looking over it again, I was able to make it work in a11c66708586ca15d5b6ee8b7dfcd422cce77459 .